OpenKeychain/src/main/java/org/sufficientlysecure/keychain/securitytoken/OpenPgpCapabilities.java

```diff
@@ -32,6 +32,7 @@
     private final static int MASK_SM = 1 << 7;
     private final static int MASK_KEY_IMPORT = 1 << 5;
     private final static int MASK_ATTRIBUTES_CHANGABLE = 1 << 2;
+    private final static int MASK_KDF_DO = 1;
```

Comment:
while we're at it, please add a comment to the [spec](https://gnupg.org/ftp/specs/OpenPGP-smart-card-application-3.4.pdf) and page (32) where those constants come from

Reply:
I've added the spec as a reference here & wherever it made sense to do so.